src/test/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCallsIntegrationTest.java

```diff
@@ -1,73 +1,209 @@
-    public void testWrongInputChunkOrder() {
-        final List<String> chunkDirectoriesNames = new ArrayList<>(Arrays.asList(
-                "/normal_cohort_chunk_2-calls/", "/normal_cohort_chunk_0-calls/", "/normal_cohort_chunk_1-calls/"));
-        final List<String> chunkDirectoriesFileList = chunkDirectoriesNames.stream()
-                .map(s -> TEST_DIR.getAbsolutePath().concat(s)).collect(Collectors.toList());
-        final File outputVCF = createTempFile("test", ".vcf");
+        /* add the rest of the required arguments if segments VCF is required */
+        if (segmentsOutputVCF != null) {
+            argumentsBuilder.addArgument(PostprocessGermlineCNVCalls.CONTIG_PLOIDY_CALLS_LONG_NAME,
+                    PLOIDY_CALLS);
+            modelShards.forEach(modelDir -> argumentsBuilder.addArgument(
+                    PostprocessGermlineCNVCalls.MODEL_SHARD_PATH_LONG_NAME, modelDir));
+            argumentsBuilder.addArgument(PostprocessGermlineCNVCalls.OUTPUT_SEGMENTS_VCF_LONG_NAME,
+                    segmentsOutputVCF.getAbsolutePath());
+        }
+        argumentsBuilder.addArgument("verbosity", "DEBUG");
 
-        final List<String> arguments = new ArrayList<>();
-        chunkDirectoriesFileList.forEach(dir -> {
-            arguments.add("-" + PostprocessGermlineCNVCalls.CHUNK_PATH_LONG_NAME); arguments.add(dir);
-        });
-        arguments.add("-" + PostprocessGermlineCNVCalls.SAMPLE_DIRECTORY_LONG_NAME);
-        arguments.add(SAMPLE_NAME);
-        arguments.add("-" + StandardArgumentDefinitions.OUTPUT_LONG_NAME);
-        arguments.add(outputVCF.getAbsolutePath());
+        if (dryRun) {
+            argumentsBuilder.addArgument(PostprocessGermlineCNVCalls.DRY_RUN_LONG_NAME, "true");
+        }
+
+        runCommandLine(argumentsBuilder.getArgsList());
+    }
+
+    @Test(dataProvider = "differentValidInput", groups = {"python"})
+    public void testDifferentValidInputIntervalsComplete(final List<String> callShards,
+                                                         final List<String> modelShards) throws IOException {
+        for (int sampleIndex = 0; sampleIndex < NUM_TEST_SAMPLES; sampleIndex++) {
+            final File actualIntervalsOutputVCF = createTempFile("intervals-output-vcf-" + sampleIndex, ".vcf");
+            final File actualSegmentsOutputVCF = createTempFile("segments-output-vcf-" + sampleIndex, ".vcf");
+            final File expectedIntervalsOutputVCF = INTERVALS_VCF_CORRECT_OUTPUTS.get(sampleIndex);
+            final File expectedSegmentsOutputVCF = SEGMENTS_VCF_CORRECT_OUTPUTS.get(sampleIndex);
+            runToolForSingleSample(callShards, modelShards, sampleIndex,
+                    actualIntervalsOutputVCF, actualSegmentsOutputVCF,
+                    ALLOSOMAL_CONTIGS, AUTOSOMAL_REF_COPY_NUMBER, false);
+            IntegrationTestSpec.assertEqualTextFiles(actualIntervalsOutputVCF, expectedIntervalsOutputVCF);
```

Comment:
Hmm, interesting.  I've used `FileUtils.contentEquals` in the past.  I'd probably stick with that to avoid any interaction with `IntegrationTestSpec.assertEqualTextFiles`, but I leave it up to you.